### PR TITLE
Canvas and mouse draw

### PR DESCRIPTION
**Pull request by zyhou**
### Simple mouse draw
presenter-mode

### Multiple color line
presenter-mode1

### Multiple size line
presenter-mode

### Fill color
presenter-mode

### Design
image

### CODE REVIEW

File: components/Canvas.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+        const canvas = canvasRef.current;
+        canvas.addEventListener('mousedown', startPaint);
+        return () => {
+            canvas.removeEventListener('mousedown', startPaint);
+        };
+    });
+
+    //Draw the line on mouse move
+    React.useEffect(() => {
```
Comment (HALLERPierre) [inline components/Canvas.js:144]: I don't understant why these 3 effects don't have dependencies list ? 
Could it be called only at mount ?

Comment (zyhou) [reply components/Canvas.js:105]: Hep, fix it :)

**Comment by Euregan** [comment]
#### 📝Changed routes:


| [**/draw**](https://oeaki-git-canvas-draw.euregan.now.sh/draw) | [**/**](https://oeaki-git-canvas-draw.euregan.now.sh/) |
|:-:|:-:|
|<a href="https://oeaki-git-canvas-draw.euregan.now.sh/draw">Screenshot of /draw</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/ArcZVbhqyQplM7R6_0Nj4dbUkxM_.png">(view full size)</a> |<a href="https://oeaki-git-canvas-draw.euregan.now.sh/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/aoZq9g3qrlRuTcwT0Rehlpk5hk8_.png">(view full size)</a> |







Commit e461ae435029253fd507abac0fff09f06e4d99d3 (https://oeaki-git-canvas-draw.euregan.now.sh).

File: pages/_app.js [hunk 2]
```diff
@@ -14,7 +16,7 @@ const App = ({ Component }) => {
             window.pool = new Pool(`${window.location.protocol === 'http:' ? 'ws' : 'wss'}://${window.location.host}`);
             setPool(pool);
```
Comment (HALLERPierre) [inline pages/_app.js:17]: I think pool will always be null but maybe I'm missing something ?

Comment (zyhou) [reply pages/_app.js:17]: Yes didn't see in first PR. Nice catch.

In future PR, the code will be remove from the `_app.js` file.

**Comment by zyhou** [comment]
All agree with my code

**Pull request merged**